knowledge-base/how-to-track-different-devices.md

```diff
@@ -0,0 +1,63 @@
+---
+title: Capture traffic from multiple devices
+description: "Learn how to capture traffic from multiple devices and how to differentiate the traffic based on the device IP."
+type: how-to
+slug: fiddler-multiple-devices
+publish: true
+res_type: kb
+---
+
+## Environment
+
+|   |   |
+|---|---|
+| Fiddler Everywhere | 1.0.0 and above |
+| Operating System | Windows, macOS, Linux |
+
+## Description
+
+Fiddler Everywhere can simultaneously capture traffic from multiple remote computers and devices (like Android and iOS smartphones).  What configuration is needed for this to happen, and how can we differentiate the traffic from each different device?
+
+## Solution (for remote computers)
+
+**Prerequisites**
```

Comment:
In general, don't use bold as a sub-section title, you can use the next heading level, for example.
In this case, I would remove the Prerequisites title because using it will require a second title for the section with the numbered list. See the suggestion below.
```suggestion
```